apps/admin-gui/src/app/shared/components/dialogs/update-application-form-dialog/update-application-form-dialog.component.html

```diff
@@ -5,10 +5,30 @@
   <div *perunWebAppsLoader="loading; indicator: spinner">
     <h1 mat-dialog-title>{{'DIALOGS.UPDATE_APPLICATION_FORM.TITLE' | translate}}</h1>
     <div class="dialog-container" mat-dialog-content>
-      <mat-form-field>
-        <mat-label>{{'DIALOGS.UPDATE_APPLICATION_FORM.MODULE_NAME' | translate}}</mat-label>
-        <input matInput [(ngModel)]="moduleName" />
-      </mat-form-field>
+      <div
+        *ngFor="let moduleName of moduleNames; let i = index; trackBy: trackByIndex"
+        class="d-flex flex-row align-items-center">
+        <mat-form-field class="w-100 mb-n2">
+          <mat-label>{{'DIALOGS.UPDATE_APPLICATION_FORM.MODULE_NAME' | translate}}</mat-label>
+          <input [(ngModel)]="moduleNames[i]" matInput />
```

Comment:
theoretically the character `,` should not be allowed in the input fields, since the backend uses it to split the module names. However it should not  be a problem since module names don't usually contain it anyways. (But it causes a slightly weird behaviour: 
![Screenshot from 2023-09-20 13-21-39](https://github.com/CESNET/perun-web-apps/assets/97669007/dbce6956-6d7e-476a-ad25-044d2ed98769)
Before refresh of the page 
![Screenshot from 2023-09-20 13-22-02](https://github.com/CESNET/perun-web-apps/assets/97669007/55664635-6aea-4964-80d9-e5137b5a15f9)
after refresh